### ISSUE TITLE
Cover image ratio/format

### ISSUE THREAD

**Issue by FF-Flo** [theCrag/website#4003]
When choose the focus of a cover image it would be really helpful when the height-width-ratio would already be fixed to the used ratio, so that you can only change position and size.

**Comment by matozoid** [comment]
Yes, in my experience the area chosen for the webcover has very little to do with the resulting webcover :-D

**Comment by scd** [comment]
Web covers have different dimensions depending on where they are used.

**Comment by FF-Flo** [comment]
Ah okay. Then what about instead showing 1 rectangle better show a form consisting of 2 overlapping rectangles, 1 with the maximum width and 1 with the maximum height? (In the way like it is calculated internally, i guess the center point is equal in every format?) So the user could see what area is shown in different formats.

**Comment by scd** [comment]
I agree we should somehow show what it looks like in standard use case
dimensions. Also we could experiment with different algorithms.

It will be a whole before we can assign resources on to this

**Comment by lordyavin** [comment]
+1
